src/components/Button/button.module.scss

```diff
@@ -42,6 +73,29 @@
     }
   }
 
+  &.Button--ghost {
+    padding: 0;
```

Comment:
Do we need the "padding : 0" here? This value is for default.

Reply:
@di-kinosian this was a requirement